packages/experimental/src/components/WebTerminal/WebTerminal.js

```diff
@@ -0,0 +1,122 @@
+import React, { useCallback, useMemo } from 'react';
+import { Close16 as Close, Help16 as Help } from '@carbon/icons-react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { pkgPrefix } from '../../global/js/settings';
+
+const WebTerminal = ({
+  open,
+  closeTerminal,
+  documentationLinks,
+  children,
+  className,
+}) => {
+  /**
+   * Triggers whenever the user clicks on an item in the help dropdown
+   */
+  const onDocumentationLinkClick = useCallback((event, onClick) => {
+    // Runs the function provided by the user if it exists
+    if (typeof onClick === 'function') {
+      // Passes the event object incase the developer wants to event.preventDefault() the link redirect
+      onClick(event);
+    }
+  }, []);
+
+  const showDocumentationLinks = useMemo(() => documentationLinks.length > 0, [
+    documentationLinks,
+  ]);
+
+  return open ? (
```

Comment:
i think it's a little cleaner to do something like this
```
if (!open) return;
return <div>...</div>
```
this way you don't have to wrap everything in the if statement

Reply:
awesome, I had it like that at first and changed it to this. I'll make the change